tensorboard/webapp/widgets/data_table/data_table_test.ts

```diff
@@ -255,4 +253,32 @@ describe('data table', () => {
       order: SortingOrder.DESCENDING,
     });
   });
+
+  it('keeps sorting arrow invisible unless sorting on that header', () => {
+    const fixture = createComponent({
+      headers: [ColumnHeaders.VALUE, ColumnHeaders.RUN, ColumnHeaders.STEP],
+      sortingInfo: {header: ColumnHeaders.VALUE, order: SortingOrder.ASCENDING},
+    });
+    fixture.detectChanges();
+    const headerElements = fixture.debugElement.queryAll(By.css('th'));
+
+    expect(
+      window.getComputedStyle(
+        headerElements[1].queryAll(By.css('.sorting-icon-container'))[0]
+          .nativeElement
+      ).opacity
+    ).toBe('1');
+    expect(
+      window.getComputedStyle(
+        headerElements[2].queryAll(By.css('.sorting-icon-container'))[0]
+          .nativeElement
+      ).opacity
+    ).toBe('0');
+    expect(
+      window.getComputedStyle(
+        headerElements[3].queryAll(By.css('.sorting-icon-container'))[0]
+          .nativeElement
+      ).opacity
```

Comment:
I probably will just check if the class "opaque" exits

Reply:
This way seems more accurate. For instance I just changed the class to "show" instead of "opaque" and this test still works. How strongly do you feel about this?

Reply:
In my understanding, those values related to visual factors will go to screenshot tests. `window.getComputedStyle` is making things more complex than needed. Instead of convinced by the example you provided, it actually concerns me more. The feeling of "something we are changing significantly does not fail the test" does not feel right to me.

I cannot see to find any usage of css and test that css attribute in current codebase. 
On the opposite I can find some examples for checking class not css value: instead of checking border-color this only checks the class:
https://cs.opensource.google/tensorflow/tensorboard/+/master:tensorboard/webapp/widgets/content_wrapping_input/content_wrapping_input_test.ts;l=146?q=tensorboard%2Fwebapp%2Fwidgets%2Fcontent_wrapping_input%2Fcontent_wrapping_input_test.ts&ss=tensorflow%2Ftensorboard

I will be convinced if you find an example OR I can still live with it if you strongly prefer this.

Reply:
The example was supposed to point out that the test I had tests what is actually important(that the user can or cannot see the icon). The name of the class is not what is important.  However since this is not tested this way elsewhere I am perfectly fine staying consistent with other tests. I have changed it.